client/Elements/CreateTopic/StepMessageRetention.patternfly.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import React, { useState } from 'react';
+import '@patternfly/react-core/dist/styles/base.css';
+import {
+  Flex,
+  FlexItem,
+  Form,
+  FormGroup,
+  Radio,
+  Select,
+  SelectOption,
+  SelectVariant,
+  TextContent,
+  Text,
+  TextVariants,
+  Touchspin,
+} from '@patternfly/react-core';
+
+interface IStepMessageRetention {
+  setMsgRetentionValue: (value: string) => void;
+}
+
+export const StepMessageRetention: React.FC<IStepMessageRetention> = ({
+  setMsgRetentionValue,
+}) => {
+  const [radio1Step3, setRadio1Step3] = useState(false);
+  const [radio2Step3, setRadio2Step3] = useState(false);
+  const [radio3Step3, setRadio3Step3] = useState(false);
+  const [radio4Step3, setRadio4Step3] = useState(false);
```

Comment:
Are there better names than is (ie dayRetention, weekRetention, etc)?

Reply:
Done made updates as radioBtnDay, radioBtnWeek, radioBtnMonth, radionBtnCustomTime